apps/epic-web/src/inngest/functions/sanity/product/sanity-product-deleted.ts

```diff
@@ -0,0 +1,132 @@
+import {inngest} from 'inngest/inngest.server'
+import {SANITY_WEBHOOK_EVENT} from 'inngest/events/sanity'
+import {prisma} from '@skillrecordings/database'
+import {stripe} from '@skillrecordings/commerce-server'
+
+export const sanityProductDeleted = inngest.createFunction(
+  {id: `product-delete`, name: 'Deactivate Product in Database'},
+  {
+    event: SANITY_WEBHOOK_EVENT,
+    if: 'event.data.event == "product.delete"',
+  },
+  async ({event, step}) => {
+    const {productId} = event.data
+
+    if (!productId) {
+      throw new Error(`Product id not found`)
+    }
+
+    const product = await step.run('get product in database', async () => {
+      return await prisma.product.findUnique({
+        where: {
+          id: productId,
+        },
+        include: {
+          prices: true,
+        },
+      })
+    })
+
+    if (!product) {
+      throw new Error(`Product not found [${productId}]`)
+    }
+
+    await step.run('deactivate product', async () => {
+      return await prisma.product.update({
+        where: {
+          id: productId,
+        },
+        data: {
+          status: 0,
```

Comment:
instead of actually deleting items in the database, this sets them to `active: 0` 

this will make sure we aren't breaking already purchased products

Reply:
I **did NOT** remove the upgrades, that might be trickier to think about...